js/ui/overview.js

```diff
@@ -533,6 +533,9 @@ const Overview = new Lang.Class({
         if (this.isDummy)
             return;
 
+        if (Main.wm.windowsAreMappedAndVisible())
```

Comment:
Can you not use `Main.workspaceMonitor.hasActiveWindows()` which is used elsewhere in `overview.js` for similar purposes?